scripts/report.py

```diff
@@ -211,7 +211,7 @@ def sonde_info_from_ipfs(flight_id):
         "launch_time": pd.Timestamp(d["launch_time"].values).to_pydatetime(warn=False),
         "platform": "HALO",
         "sonde_id": d.attrs["SondeId"],
-        "flag": "UNKNOWN",
+        "flag": "ALL_FLAGS",
```

Comment:
What does it mean if the sonde flag is `ALL_FLAGS`?

Reply:
So far we don't split between sondes of different flags. We might do so as soon as the Level_3 and QC for the dropsondes is finished and we have reasonable dropsonde flags. Currently, we display/list sondes of all flags (GOOS, BAD, UGLY). I simply exchanged the name "UNKNOWN" by "ALL_FLAGS" to make it more intuitive. I found it hard to understand before what the number/list actually means.